docs/classes/request.md

```diff
@@ -2,86 +2,128 @@
 title: Request
 permalink: /classes/request
 ---
+# Request
+
+Class **`Request`** extends [`Message`](message.md)
```

Comment:
This should only be an internal detail IMO, maybe we should make `Message` a trait?

Reply:
Removed.